rust/noosphere-core/src/view/sphere.rs

```diff
@@ -544,6 +546,29 @@ impl<S: BlockStore> Sphere<S> {
             Authorization::Ucan(ucan),
         ))
     }
+
+    /// Get a [Stream] that yields the [ChangelogIpld] for content slugs at each
+    /// version of the sphere.
+    pub fn changes_since(
+        self,
+        version: Option<&Cid>,
+    ) -> impl Stream<Item = Result<(Cid, ChangelogIpld<MapOperation<String, Cid>>)>> {
+        let version = version.cloned();
+
+        try_stream! {
```

Comment:
:eyes: I'll need this in the future, great illustration of stream

Reply:
I really love using streams in Rust. But, they come with a DX downside: their interface on the implementation side is verbose, and Rust Analyzer is really bad at providing static analysis of any code inside the `stream!` and `try_stream!` blocks (which are necessary to make usage of streams readable to humans).